### ISSUE TITLE
A task has been opened in the USWDS repo

### ISSUE THREAD

**Issue by msbtterswrth** [department-of-veterans-affairs/tmf-auth-exp-design-patterns#97]
### Background
Now that we have some initial guidance for contributing to USWDS, we want to open a task for our first pattern, "Know when their information is prefilled".

### ACs
- [x] We have reached out to Liz Lantz and other GSA members to determine what needs to be included in our USWDS issue
- [x] An issue has been opened in the [USWDS repo](https://github.com/uswds/uswds/issues)
- [x] We have notified our contacts at GSA that we've opened a task for our pattern so they can expedite it's review

**Comment by andaleliz** [comment]
note to self 😁  Look into form related patterns within USWDS, do some digging into how other patterns have been contributed/framed, in response to this thought from Slack:

>I'd love to get your feedback on how our pattern may generally fit within USWDS, and how we should frame it (we noticed there's no code really in USWDS, as opposed to VADS)

@msbtterswrth now that I think on it, when you say "how we should frame it" do you mean in terms of pitching it to USWDS for acceptance into the system, or in terms of writing up guidance for end users to consume and be able to use the pattern. Or both? Any more clarification would be welcomed!

**Comment by msbtterswrth** [comment]
@andaleliz great question, I think I meant more of how should we write up the guidance, we know that patterns there are nested, and we assume this one should live under the 'complex forms' pattern, but we also noticed most of the patterns at USWDS do not include code or components, only guidance. Should we include our recommended code as well or just the governance piece?

**Comment by bellepx0** [comment]
For reference: [Slack thread](https://vhahelpdesks.slack.com/archives/C07CTUVKJGP/p1724780573343899) with USWDS about how to contribute our pattern. 
TLDR; Code is helpful but not required. Code could also be from our own repo and we link to it in our proposal.

**Comment by bellepx0** [comment]
Created a draft proposal [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/8942cf87a3a161c74a6bcc72e9cf1d059e1888bc/products/authenticated-patterns/patterns/prefill/%2397-Draft%20USWDS%20Prefill%20Pattern%20Issue.md)

**Comment by msbtterswrth** [comment]
In the monday meeting, we discussed opening this task under the github discussions rather than the issues, in order to get more feedback from the community.

**Comment by bellepx0** [comment]
Proposal for prefill pattern has been submitted [here](https://github.com/uswds/uswds/discussions/6053)

**Comment by msbtterswrth** [comment]
Exceptional work!

**Comment by beckyphung** [comment]
Lgtm. Apologies closing this a day after sprint ended. This will still count towards Sprint 6.
